common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/ViewExtension.kt

```diff
@@ -148,3 +148,9 @@ fun Slider.quietlySetValue(
     value = newValue
     addOnChangeListener(listener)
 }
+
+fun View.disable() {
```

Comment:
We usually decide how `disable` items look using styles. Any reason why we have decided to do this programatically?

Reply:
It's for a recycleView menu item which needs to be programatically. If list of websites is empty, then overflow menu item "Remove All" is disabled, if we add a website it will be enabled.

Reply:
Discussed on zoom and we can remove this method and directly use `setEnabled = false` on that `TextView`.